eta/core/video.py

```diff
@@ -1993,6 +1981,7 @@ def __init__(self, inpath, frames=None, keyframes_only=False):
         self._ffmpeg = FFmpeg(
             in_opts=in_opts,
```

Comment:
I noticed above that had previously added the `-vsync 0` option as an _input option_ when the `keyframes_only` option is turned on.  I'm wondering if there are any other circumstances where this would be appropriate as an input option?